vignettes/replication.Rmd

```diff
@@ -0,0 +1,111 @@
+---
+title: "Replication by `rdlearn`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{my-vignette}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+knitr::opts_chunk$set(eval = TRUE)
+```
+
+## Introduction
+
+The R package `rdlearn` implements the *safe policy learning under regression discontinuity designs with multiple cutoffs* of Zhang et al. (2022). It provides functions to learn treatment assignment rules (cutoffs) which are guaranteed to yield no worse overall outcomes than the existing cutoffs.
+
+In this vignette, we replicate some of the empirical application results from Chapter 6 of Zhang et al. (2022).
+
+## Installation
+
+The `rdlearn` package for R can be downloaded using (requires previous installation of the [`remotes` package](https://remotes.r-lib.org/index.html)).
+
+```{r, eval = FALSE}
+remotes::install_github("kkawato/rdlearn")
```

Comment:
If this vignette is intended for reproducing the results in the paper, consider adding an explicit reference to the version of the package. This prevents the vignette contents updated in the future when the package version is updated.

https://github.com/r-lib/remotes/blob/main/R/install-github.R#L11-L15